gitlab/mixins.py

```diff
@@ -367,7 +342,13 @@ def update(
         else:
             path = f"{self.path}/{utils.EncodedId(id)}"
 
-        self._check_missing_update_attrs(new_data)
+        if self._obj_cls is not None:
+            excludes = [self._obj_cls._id_attr]
+        else:
+            excludes = None
```

Comment:
Would be better to make sure nothing is `None`

```suggestion
        excludes = []
        if self._obj_cls is not None and self._obj_cls._id_attr is not None:
            excludes = [self._obj_cls._id_attr]
```

Reply:
done.

Reply:
forgot to push